src/pkg/fault/fault.go

```diff
@@ -24,6 +24,11 @@ type Bus struct {
 	// we'd expect to see 1 error added to this slice.
 	recoverable []error
 
+	// skipped is the accumulation of skipped items.  Skipped items
+	// are not errors themselves, but instead represent some permanent
+	// inability to process an item, due to a well-known cause.
```

Comment:
Instead of redefining Skip in several locations, point back to the details in some way.
`@see /src/pkf/fault/fault.go:Skip` or whatever the idiomatic go way is for comments.

Reply:
Eh, that's not very user friendly.  I would not, myself, want to look up other comments by reference.  The idiomatic way is to have the necessary information available where it's needed.

And we can always adjust this in the future, if wanted.